www/src/css/lib/vanilla-overrides.scss

```diff
@@ -5,4 +5,50 @@
 // If any styles from vanilla framework need to be overridden, corresponding
 // SCSS rules should be put in this file (not mixed with other styles).
 
+// scss/modules/_header.scss
+.banner .nav-primary ul li:hover {
+  background-color:#f7f7f7;
+}
 
+.banner .nav-primary ul li a,
+.banner .nav-primary ul li a:link,
+.banner .nav-primary ul li a:visited {
+  background-color:transparent;
+  font-size:initial;
+  border-left:0;
+  padding:14px 12px 8px 12px;
+}
+
+.banner .nav-primary ul li a:hover,
+.banner .nav-primary ul li a:link:hover,
+.banner .nav-primary ul li a:visited:hover {
+  background-color:transparent;
+}
+
+@media screen and (max-width: 640px) {
+  .banner .logo {
+    margin:0 0 0 10px;
+  }
+
+  .banner .nav-primary ul {
+    background-color:transparent;
+    display:block;
+	border-top:0;
+  }
+
+  .banner .nav-primary ul li {
+    width:auto;
+    border-right:0;
+    border-left:0;
+  }
+}
+
+@media screen and (min-width: 640px) {
+  .banner .nav-primary span {
+    display:initial;
+  }
+
+  .banner .logo {
+    margin:0 0 0 20px;
```

Comment:
`.banner .logo` margin styles - here and above (20px or 10px) seem not very different from what vanilla has by default. Do they need to be overridden at all? Won't defailt vanilla margins be good enough?